### PR TITLE
Creates LastModifiedDateTime field in Person

### LINKED ISSUE #110: Create Class & field in Person

### PR DESCRIPTION
**Pull request by xavierpok**
Creates the corresponding field and relevant methods, as well as supporting JSON serialisation for this field.

Initialisation with meaningful values as edit/creation time is not supported (yet)

Passes all test cases in PR, including new/edited test cases.

Currently, any `Person` object instantiated through normal (I.e. the `add` or `edit` command) will initialise the last modified to 10 Oct 2020 10:10:00

Closes #110

### CODE REVIEW

**Review by geoffong11 — APPROVED** [review]
LGTM!

File: src/main/java/seedu/address/model/person/Person.java [hunk 1]
```diff
@@ -26,8 +26,9 @@ public class Person {
     private final Job job;
     private final Set<Tag> tags = new HashSet<>();
 
+    private final LastModifiedDateTime lastModifiedDateTime;
     /**
-     * Every field must be present and not null.
+     * Every field must be present and not null
```
Comment (imkwokyong) [inline src/main/java/seedu/address/model/person/Person.java:31]: Maybe a full stop here

File: src/test/java/seedu/address/storage/JsonAdaptedPersonTest.java [hunk 2]
```diff
@@ -26,6 +27,7 @@ public class JsonAdaptedPersonTest {
     private static final String INVALID_EMAIL = "example.com";
     private static final String INVALID_TAG = "#friend";
 
+    private static final String INVALID_LASTMODIFIEDDATETIME = "Nov 40, 2000 13:00:10 AM";
```
Comment (imkwokyong) [inline src/test/java/seedu/address/storage/JsonAdaptedPersonTest.java:30]:
```suggestion
    private static final String INVALID_LAST_MODIFIED_DATETIME = "Nov 40, 2000 13:00:10 AM";
```

**Review by imkwokyong — APPROVED** [review]
LGTM, just some naming convention and comments nits! Can change them in next PR.

**Pull request merged**